### PR TITLE
Added MacOS support on CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by maddychan**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -245,3 +245,34 @@ npm run update_prebuilt
 
 That last step is easy to forget, so try to make it a habit whenever you're
 prepping a PR that includes frontend work.
+
+### MacOS
```
Comment (jettisonjoe) [inline CONTRIBUTING.md:249]: I would move this whole thing up to around line 182, just under the Debian/Ubuntu install steps; it still belongs under the 'Setting Up Your Dev Environment' heading.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -245,3 +245,34 @@ npm run update_prebuilt
...
+
+Install Xcode and Homebrew Mac. Once that is done:
```
Comment (jettisonjoe) [inline CONTRIBUTING.md:251]: Might be good to provide a few words of explanation and make Xcode and Homebrew hyperlinks to their respective landing pages. Is Xcode just a dep for Homebrew, or do you need it in order to get git as well? If it's just for Homebrew, then we might not need to list it here.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -245,3 +245,34 @@ npm run update_prebuilt
 
 That last step is easy to forget, so try to make it a habit whenever you're
 prepping a PR that includes frontend work.
+
+### MacOS
+
+Install Xcode and Homebrew Mac. Once that is done: 
+
+```bash
+#Install dependent packages
```
Comment (jettisonjoe) [inline CONTRIBUTING.md:254]: I'd just call these "dependencies". In this case, I'd say OpenHTF is on the "dependent" side of the relation.

Comment (jettisonjoe) [inline CONTRIBUTING.md:254]: nit: To me, a space between the # and the comment text makes comments more readable.
nit+: I also like comments to be full sentences(ish) with capitalization and punctuation.

Google does have a [shell style guide](https://google.github.io/styleguide/shell.xml#Comments), but it doesn't explicitly verify my nits that I can see (though its examples are consistent with my nits).

File: CONTRIBUTING.md (hunk 2 above)
Comment (maddychan) [inline CONTRIBUTING.md:251]: I googled it and it seems that a gcc compiler is needed for homebrew and pip, and xcode is one way to get that but not the only one. What do you suggest I put?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -245,3 +245,34 @@ npm run update_prebuilt
...
+
+#Change to the openhtf directory 
+cd openhtf
+
+#Create a new virtualenv
+virtualenv venv
+
+#Activate the new virtualenv
+. venv/bin/activate
+```
```
Comment (jettisonjoe) [inline CONTRIBUTING.md:278]: Should add a `python setup.py develop` step at the end.

**Pull request closed without merging**